plugins/experimental/traffic_dump/traffic_dump.cc

```diff
@@ -38,31 +37,37 @@
 #include <chrono>
 #include <atomic>
 #include <string>
+#include <string_view>
 
+#include "tscore/ts_file.h"
 #include "ts/ts.h"
 
-const char *PLUGIN_NAME          = "traffic_dump";
-static const std::string closing = "]}]}";
-
-static std::string LOG_DIR = "dump";                // default log directory
-static int s_arg_idx       = 0;                     // Session Arg Index to pass on session data
-static std::atomic<int64_t> sample_pool_size(1000); // Sampling ratio
-
+namespace
+{
+const char *PLUGIN_NAME   = "traffic_dump";
+const std::string closing = "]}]}";
+
+ts::file::path log_path{"dump"};               // default log directory
+int s_arg_idx = 0;                             // Session Arg Index to pass on session data
+std::atomic<int64_t> sample_pool_size(1000);   // Sampling ratio
```

Comment:
I'll let it slide, but if it is required to populate these values with options, then they should be initialized to zero so you can detect whether that happened. These values are simply confusing, as they are meaningless in the larger scope.